### DISCUSSION TITLE
Darstellungsvariante Layergruppe InfoButton

### DISCUSSION THREAD

**Discussion by marko-jer** [e-netze/webgis-community#107] [General]
Der InfoButton / Metadaten Link wird bei einer Layergruppe nur angezeigt, wenn DisplayStyle auf Dropdown gestellt ist. 
Eine Darstellung des InfoButtons / Metadaten Links unabhängig vom DisplayStyle wäre wünschenswert.

**Comment by gis-eni** [comment]
Ist mit Build 5.23.3901 behoben
